src/en/authors-charm-icon.md

```diff
@@ -2,36 +2,22 @@ Title: Creating icons for charms
 
 # Creating icons for charms
 
-In order to make your charm recogniseable in the Charm Store, it is best
-practice to add a unique and recogniseable icon for it. This isn't as hard as it
-may seem, as this step by step guide will demonstrate:
+In order to make your charm recognizable in the Charm Store, it is best
```

Comment:
That was a typo, but it should be "recognisable"

Reply:
afaik, both are correct, it's US vs UK English. My spell checker must be on US English..

Reply:
Yes, the 'z' spelling would be US spelling. All the docs are supposed to be UK English, though I think a few zs have slipped through. Thanks again!